flicker/flicker.py

```diff
@@ -204,6 +204,21 @@ def _reset(self, df):
         self._nrows = None
         self._ncols = None
 
+    def _validate_column_name(self, name, names=None, dataframe_name=None):
+        if not isinstance(name, six.string_types):
+            msg = ('column name must be of type str but you provided '
+                   'type = {}').format(str(type(name)))
+            raise TypeError(msg)
+        if names is None:
+            names = self._df.columns
+        if name not in names:
+            if dataframe_name is None:
+                msg = 'column name "{}" not found'
```

Comment:
I think you're missing a `msg.format(name)` for this?

Reply:
You're right. Thanks!